src/style.css

```diff
@@ -1 +1,42 @@
-/* add styles here */
+.page {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+  width: fit-content;
```

Comment:
It's a good practice not to fix container sizes unless required. Here, the width is set to `fit-content`, which is good as it allows the content to dictate the size of the element. Just ensure this is the desired behavior in your layout.